controller.py

```diff
@@ -339,7 +339,9 @@ def validate_email(self):
             "help_email": Hyperlink.HELP_REL,
             "copyright_email": Hyperlink.COPYRIGHT_DESIGNATED_AGENT_REL
         }
-        hyperlink = Library.get_hyperlink(library, email_types[email])
+        hyperlink = None
+        if email_types[email]:
```

Comment:
This will still crash if `email` isn't a key in `email_types`. Try `email_types.get(email)` instead.